opensir/models/model.py

```diff
@@ -39,9 +39,14 @@ def __init__(self):
         self.p = None
         self.pop = None
         self.w0 = None
-        self.pcov = None
-        self.fit_input = None
-        self.fit_index = None  # Store fitting info for post-regression analysis
+        # Attributes setted after fitting
+        self.fit_attr = {
```

Comment:
I guess it's better here to define "self.fit_attr" to None. Then, the "fit" function creates the dictionary and assigns it directly.

Reply:
Alright. The fit function will create the dictionary. I noticed that there was one attribute that doesn't depends on the fit, but it depends on the model, and it is fit_input. I included it on the model attributes again.